### ISSUE TITLE
Logging in Production

### ISSUE THREAD

**Issue by bradennapier** [olalonde/follow-redirects#89]
Not really sure why this module is logging in production but its costing a LOT of money in cloud watch ... NODE_ENV is = production but even if it weren't not sure why this random module that is a deep dependency in the project is sitting herel ogging some large object millions of times in my application

```
Thu, 12 Jul 2018 18:36:10 GMT follow-redirects options { ... }
```

**Comment by bradennapier** [comment]
Ok... i do see you commented on this.. not really sure what set the env var DEBUG to * ... will check

**Comment by RubenVerborgh** [comment]
> not really sure what set the env var DEBUG to *

Let us know if you find out, hope this random module is still useful to you :wink:

**Comment by bradennapier** [comment]
axios sure is ;-) just extremely irritating when random logs show up from a module you didnt even know about heh.  It was AWS Lambda addig the env var - not sure yet where in the process - i overwrote it for now to stop the logs - at 3 billion requests a month that much data being logged can be.... expensive (around $80k)

**Comment by RubenVerborgh** [comment]
Myeah, blame Amazon. We're just following best practices. Beyond the economic aspect, I don't see a good reason why they would turn on debugging by default.

**Comment by bradennapier** [comment]
Nah, this is def something one of our team members did at a higher level.  Luckily we caught it almost immediately heh.  Still costed a few hundred. 

tl;dr

## Setup CloudWatch Alarms :-)
